### PR TITLE
Create august-2024-newsletter.md

### PR DESCRIPTION
**Pull request by LEQUI000**
Creating the August 2024 Newsletter mirror which details information on upcoming events in August, subscribers will be sent a similar but not identical version to the website.

### CODE REVIEW

File: _posts/august-2024-newsletter.md
Comment (CurtisGreen) [inline _posts/august-2024-newsletter.md]: Is this intended to be written like it's a letter from someone?

File: _posts/august-2024-newsletter.md
Comment (CurtisGreen) [inline _posts/august-2024-newsletter.md]: Either should have no comma or should have someone's name or "DATA", or whatever

File: _posts/august-2024-newsletter.md
Comment (CurtisGreen) [inline _posts/august-2024-newsletter.md]:
```suggestion
The first meeting is the DART Board meeting at [DART Headquarters](https://maps.app.goo.gl/36uCk3suvRdJPfM47) on August 27, 2024 at 6pm, please arrive 30 minutes early to register for public comments.
```

File: _posts/august-2024-newsletter.md
Comment (CurtisGreen) [inline _posts/august-2024-newsletter.md]: I think opposing cuts should be 1. rather than more service

File: _posts/august-2024-newsletter.md
Comment (LEQUI000) [reply _posts/august-2024-newsletter.md]: Originally yes, this was pulled directly from the email we are sending out. I'm just now realizing it should be slightly more formal

File: _posts/august-2024-newsletter.md
Comment (CurtisGreen) [inline _posts/august-2024-newsletter.md]: I think for each of these the meeting info should be laid out and then a brief description of them. That lets people see the dates at a glance and increases the likelihood they'll read it

File: _posts/august-2024-newsletter.md
Comment (LEQUI000) [inline _posts/august-2024-newsletter.md:45]: Remove the extra space here

File: _posts/august-2024-newsletter.md
Comment (LEQUI000) [inline _posts/august-2024-newsletter.md:51]: Remove the extra space here

File: _posts/august-2024-newsletter.md
Comment (CurtisGreen) [inline _posts/august-2024-newsletter.md]:
```suggestion
Registration will open on the 22nd at 8:15 am, you can register to speak 3 ways.
```

File: _posts/august-2024-newsletter.md
Comment (LEQUI000) [inline _posts/august-2024-newsletter.md]: Delay sending until tomorrow update the date to August 9

File: _posts/august-2024-newsletter.md
Comment (LEQUI000) [reply _posts/august-2024-newsletter.md]: Do you have some proposed wording?

File: _posts/august-2024-newsletter.md
Comment (LEQUI000) [reply _posts/august-2024-newsletter.md]: Agreed

File: _posts/august-2024-newsletter.md
Comment (LEQUI000) [reply _posts/august-2024-newsletter.md]: Changed to less of a letter

File: _posts/august-2024-newsletter.md
Comment (democat3457) [inline _posts/august-2024-newsletter.md]:
```suggestion
First, our August General Meeting will be on August 18, 2024, from 3-5pm at The Metropolitan located at [1200 Main Street, Dallas](https://maps.app.goo.gl/pPBSUsQeJYZr1hxf7). When you get there, enter via the Murphy Drive entrance and head to the community room on the 6th floor (same room as the first general meeting). We will have light snacks and refreshments available.
```

File: _posts/august-2024-newsletter.md
Comment (democat3457) [inline _posts/august-2024-newsletter.md]:
```suggestion
The DART Board meeting is at [DART Headquarters](https://maps.app.goo.gl/36uCk3suvRdJPfM47) at Akard Station on August 27, 2024, at 6pm. Please arrive 30 minutes early to register for public comments.
```

File: _posts/august-2024-newsletter.md
Comment (democat3457) [inline _posts/august-2024-newsletter.md]:
```suggestion
Some members of the DART board are not very supportive of transit. Right now, there is a push among some board members to cut DART’s operating budget by 5-10% this year and cap DART’s revenue growth. Other DART board members have expressed willingness to compromise with cities that want to cut DART’s funding.
```

File: _posts/august-2024-newsletter.md
Comment (democat3457) [inline _posts/august-2024-newsletter.md]:
```suggestion
By showing up, we wish to communicate these points to the DART board:
```

File: _posts/august-2024-newsletter.md
Comment (democat3457) [inline _posts/august-2024-newsletter.md]: draconian is a weird adjective to use

File: _posts/august-2024-newsletter.md
Comment (democat3457) [inline _posts/august-2024-newsletter.md:24]: Horizontal lines shouldn't be used here or anywhere else, probably

File: _posts/august-2024-newsletter.md
Comment (CurtisGreen) [reply _posts/august-2024-newsletter.md]: It’s okay if it’s like this but I think it would need to be consistent all the way though

File: _posts/august-2024-newsletter.md
Comment (democat3457) [inline _posts/august-2024-newsletter.md]:
```suggestion
The Dallas City Council meeting is at [City Hall](https://maps.app.goo.gl/BgTdpbSjKRe73pF88) on August 28, 2024, at 9am. We intend to make a big showing here in support of DART funding, and we wish to communicate that we are:
```

File: _posts/august-2024-newsletter.md
Comment (democat3457) [inline _posts/august-2024-newsletter.md]:
```suggestion
Registration for public comments will close at 5pm on August 27th. If you plan to attend this meeting in-person, please let us know so we can get a count. The City of Dallas also allows speaking virtually as an option in case you cannot attend in-person.
```

File: _posts/august-2024-newsletter.md
Comment (democat3457) [inline _posts/august-2024-newsletter.md]:
```suggestion
Registration to speak at the City Council meeting will open on August 22nd at 8:15 am. There are 3 ways to register:
```

File: _posts/august-2024-newsletter.md
Comment (democat3457) [inline _posts/august-2024-newsletter.md]:
```suggestion
* [DART Headquarters](https://maps.app.goo.gl/36uCk3suvRdJPfM47) at Akard Station
```

File: _posts/august-2024-newsletter.md
Comment (CurtisGreen) [inline _posts/august-2024-newsletter.md]:
```suggestion
* August 18, 2024 from 3pm to 5pm
```

File: _posts/august-2024-newsletter.md
Comment (democat3457) [inline _posts/august-2024-newsletter.md]:
```suggestion
```

File: _posts/august-2024-newsletter.md
Comment (democat3457) [inline _posts/august-2024-newsletter.md]:
```suggestion
1. Register online at [personalfinance.dallascityhall.com/speakers.aspx](https://personalfinance.dallascityhall.com/speakers.aspx).
```

File: _posts/august-2024-newsletter.md
Comment (democat3457) [inline _posts/august-2024-newsletter.md]:
```suggestion
2. Email [secspeakerrequest@dallas.gov](mailto:secspeakerrequest@dallas.gov) with your name, address, phone number, and meeting method (teleconference or in-person).
```

File: _posts/august-2024-newsletter.md
Comment (democat3457) [inline _posts/august-2024-newsletter.md]:
```suggestion
3. Call the office at [214-670-3738](tel:+12146703738) with your name, address, phone number, and meeting method (teleconference or in-person).
```

File: _posts/august-2024-newsletter.md
Comment (democat3457) [inline _posts/august-2024-newsletter.md]:
```suggestion
 > NOTE: The City of Dallas limits speakers to one public comment per month, so please be wary of that. You can read the full policy on the [Dallas City Council website](https://dallascityhall.com/government/citysecretary/pages/ccrules.aspx).
```

File: _posts/august-2024-newsletter.md
Comment (democat3457) [inline _posts/august-2024-newsletter.md]:
```suggestion
As a reminder, the public meetings we plan on attending are back to back - the DART Board meeting is Tuesday evening, and the Dallas City Council meeting is Wednesday morning. If you can only attend one, Dallas residents should prioritize attending the Dallas City Council meeting.
```

File: _posts/august-2024-newsletter.md
Comment (democat3457) [inline _posts/august-2024-newsletter.md]:
```suggestion
We intend to make a big showing here in support of DART funding, and we wish to communicate that we are:
```

**Review by LEQUI000 — COMMENTED** [review]
This will probably will be the final draft

File: _posts/august-2024-newsletter.md
Comment (democat3457) [inline _posts/august-2024-newsletter.md]:
```suggestion
To get the latest updates and news, follow DATA's social media at [https://dallasareatransitalliance.org/contact](https://dallasareatransitalliance.org/contact).
```

File: _posts/august-2024-newsletter.md
Comment (democat3457) [inline _posts/august-2024-newsletter.md]:
```suggestion
* Use the Murphy Drive entrance. The meeting will be in the Community Room on the 6th floor (same room as the first general meeting).
```

File: _posts/august-2024-newsletter.md
Comment (democat3457) [inline _posts/august-2024-newsletter.md]:
```suggestion
* We will have light snacks and refreshments available.
```

**Review by democat3457 — COMMENTED** [review]
Lots of grammar fixes and rewording

File: _posts/august-2024-newsletter.md
Comment (LEQUI000) [reply _posts/august-2024-newsletter.md]: blame Connor not me :p

File: _posts/august-2024-newsletter.md
Comment (LEQUI000) [reply _posts/august-2024-newsletter.md:24]: was imported from the email but we can remove it if it doesn't look good

File: _posts/august-2024-newsletter.md
Comment (democat3457) [inline _posts/august-2024-newsletter.md]:
```suggestion
DATA's August newsletter highlights key events, including our General Meeting on August 18 and upcoming public meetings at the DART Board and Dallas City Council.
```

File: _posts/august-2024-newsletter.md
Comment (LEQUI000) [inline _posts/august-2024-newsletter.md:13]: Shouldn't this be in the excerpt?

Comment (Schobbish) [reply _posts/august-2024-newsletter.md:13]: s/DATA's/Our

Comment (democat3457) [reply _posts/august-2024-newsletter.md:13]: The other articles have the excerpt at the top of the page as well, so I figured I'd follow the format.

Comment (democat3457) [reply _posts/august-2024-newsletter.md:13]: > s/DATA's/Our

Also, I figured it would be important to dictate who "we" are at the very top of the article

**Review by CurtisGreen — APPROVED** [review]
Approving this because while there are tweaks to be made I’d rather have something ready than be stuck waiting for an approval. Can make more edits tomorrow

**Pull request merged**
